nnvm/python/nnvm/to_relay.py

```diff
@@ -0,0 +1,530 @@
+# pylint: disable=no-else-return, unidiomatic-typecheck, invalid-name, unused-argument
+"""convert nnvm graph to relay"""
+from tvm import relay, nd
+from tvm.relay import op, expr, var
+from tvm.relay.frontend.common import StrAttrsDict
+from .symbol import Symbol
+from .compiler import graph_attr
+from .graph import create as graph_create
+import numpy
+import json
+
+
+def _to_nn_batch_flatten(children, attrs, odtype):
+    assert len(children) == 1
+    return op.nn.batch_flatten(children[0])
+
+
+def _to_dense(children, attrs, odtype):
+    use_bias = attrs.get_bool('use_bias')
+    units = attrs.get_int('units')
+    dense = op.nn.dense(children[0], children[1], units=units)
+    if use_bias:
+        return op.nn.bias_add(dense, children[2])
+    else:
+        return dense
+
+
+def _to_relu(children, attrs, odtype):
```

Comment:
Likely we can use _rename in https://github.com/dmlc/tvm/blob/master/python/tvm/relay/frontend/mxnet.py#L28